generate_diagrams.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+directory="./docs/model_diagrams/"
+
+if [ ! -d "$directory" ]; then
+    # If the directory doesn't exist, create it
+    mkdir -p $directory
+fi
+
+
+SETTINGS_MODULE="$1"
+
+shift
+
+declare -A unique_apps
```

Comment:
Not used.
```suggestion
```

Reply:
d46460d83a85b249af2f081c5bb2cb4ab73faa0f